tapa/graphir_conversion/gen_rs_graphir.py

```diff
@@ -29,3 +57,499 @@ def get_verilog_module_from_leaf_task(task: Task) -> VerilogModuleDefinition:
         verilog=task.module.code,
         submodules_module_names=(),
     )
+
+
+def get_slot_module_definition_parameters(
+    leaf_modules: dict[str, VerilogModuleDefinition],
+) -> list[ModuleParameter]:
+    """Get slot module parameters."""
+    # merge parameters from leaf modules
+    parameters = {}
+    for leaf_module in leaf_modules.values():
+        for param in leaf_module.parameters:
+            if param.name not in parameters:
+                parameters[param.name] = param
+            # merge parameter
+            elif param.expr != parameters[param.name].expr:
+                _logger.warning(
```

Comment:
This should be an error instead of a warning, right? In fact, renaming the parameter is the only right choice, I assume.